bobweb/bob/command_gpt.py

```diff
@@ -126,31 +137,49 @@ def build_message(self, chat_id: int, system_prompt: str = None) -> List:
+        if sub_command_parameter.strip() == '':
+            quick_system_prompts = database.get_quick_system_prompts(update.effective_message.chat_id)
+            current_prompt = quick_system_prompts[sub_command]
+            empty_message_last_part = f" tyhjä. Voit asettaa pikaohjausviestin sisällön komennolla '/gpt {sub_command} = (uusi viesti)'."
+            current_message_msg = empty_message_last_part if current_prompt is None else f':\n\n{current_prompt}'
+            update.effective_message.reply_text(f"Nykyinen pikaohjausviesti {sub_command} on nyt{current_message_msg}")
+        else:
+            database.set_quick_system_prompt(update.effective_chat.id, sub_command, sub_command_parameter)
+            quick_system_prompts = database.get_quick_system_prompts(update.effective_message.chat_id)
+            current_prompt = quick_system_prompts[sub_command]
+            update.effective_message.reply_text(f"Uusi pikaohjausviesti {sub_command} on nyt:\n\n{current_prompt}")
+
     def handle_quick_system_prompt_sub_command(self, update: Update, command_parameter, context: CallbackContext = None):
         sub_command = command_parameter[1]
         sub_command_parameter = get_content_after_regex_match(command_parameter, quick_system_prompt_sub_command_regex)
 
         # If actual prompt after quick system prompt option is empty
         if sub_command_parameter.strip() == '':
+            quick_system_prompts = database.get_quick_system_prompts(update.effective_message.chat_id)
+            no_parameters_given_notification_msg = generate_no_parameters_given_notification_msg(quick_system_prompts)
             update.effective_message.reply_text(no_parameters_given_notification_msg)
         else:
             self.gpt_command(update, sub_command_parameter, context, system_prompt_id=sub_command)
 
-quick_system_prompts = {
-    "1": "You are an assistant named Bob (or Robert the Bot officially). Be brief and concise in your answers.",
-    "2": "Olet suomenkielinen apuri nimeltä Bob.",
-    "3": "You are an expert code developer with teaching skills."
-}
-
-no_parameters_given_notification_msg = f"""
-    Anna jokin syöte komennon
-    jälkeen. [.!/]gpt (syöte). Voit valita jonkin kolmesta
-    valmiista ohjeistusviestistä laittamalla numeron 1-3
-    ennen syötettä.\n
-    1: {quick_system_prompts["1"]}\n
-    2: {quick_system_prompts["2"]}\n
-    3: {quick_system_prompts["3"]}
-"""
+
+def generate_no_parameters_given_notification_msg(quick_system_prompts: dict = None):
+    if quick_system_prompts:
+        quick_system_prompts_str = ''.join([f'\n{key}: {value}' for key, value in quick_system_prompts.items()])
+    else:
+        quick_system_prompts_str = ''
+    no_parameters_given_notification_msg = f"""
+        Anna jokin syöte komennon
+        jälkeen. [.!/]gpt (syöte). Voit valita jonkin kolmesta
+        valmiista ohjeistusviestistä laittamalla numeron 1-3
+        ennen syötettä.
+        {quick_system_prompts_str}
+    """
```

Comment:
Tästä multiline String aiheuttaa tekstin preformatoitumisen niin, että se sisältää noi rivinvaihdot ja rivin sisennykset. Voisi korjata:

```suggestion
    no_parameters_given_notification_msg = \
        f'Anna jokin syöte komennon jälkeen. [.!/]gpt (syöte). Voit valita jonkin kolmesta valmiista ' \
        f'ohjeistusviestistä laittamalla numeron 1-3 ennen syötettä. {quick_system_prompts_str}'
```